server/src/test/java/io/crate/integrationtests/AggregateExpressionIntegrationTest.java

```diff
@@ -305,4 +308,33 @@ public void test_assure_cmp_by_function_call_with_reference_and_literal_does_not
         execute("select max_by(name, 1) from tbl;");
         assertThat(response).hasRows("foo");
     }
+
+    @SuppressWarnings("unchecked")
+    public void test_topk_agg() {
+        execute("create table tbl (l long, l_no_doc_values long storage with(columnstore = false))");
+        execute("insert into tbl(l, l_no_doc_values) values (1, 1), (1, 1), (1, 2), (2, 2), (2, 2)");
+        execute("refresh table tbl");
+
+        // Use this very verbose style of assertion, since the return type is an Array<Object(UNDEFINED)>,
```

Comment:
I could stick to `@UseJdbc(0)` and have more beautiful assertions, what do you think?

Reply:
Either way is fine by me. I'd tend to keep the assertions as is and keep jdbc enabled